README.md

```diff
@@ -1,289 +1,83 @@
 # LitElement
+A simple base class for creating fast, lightweight web components.
 
+[![Build Status](https://travis-ci.org/Polymer/lit-element.svg?branch=master)](https://travis-ci.org/Polymer/lit-element)
 [![Published on npm](https://img.shields.io/npm/v/lit-element.svg)](https://www.npmjs.com/package/lit-element)
 [![Published on webcomponents.org](https://img.shields.io/badge/webcomponents.org-published-blue.svg)](https://www.webcomponents.org/element/lit-element)
 [![Mentioned in Awesome lit-html](https://awesome.re/mentioned-badge.svg)](https://github.com/web-padawan/awesome-lit-html)
 
-## A simple base class for creating fast, lightweight web components
+## Documentation
+
+Full documentation is available at [lit-element.polymer-project.org](https://lit-element.polymer-project.org).
+
+Docs source is in the `docs` folder. To build the site youself, see the instructions in [docs/README.md](docs/README.md).
```

Comment:
I think we should move this into CONTRIBUTING.md for both projects

Reply:
Done.